python_modules/automation/automation/docker/images/dagster-base/Dockerfile

```diff
@@ -16,7 +16,10 @@ RUN \
         dagster-aws==${DAGSTER_VERSION} \
         dagster-cron==${DAGSTER_VERSION} \
 # Cleanup
-    &&  rm -rf /var \
-    &&  rm -rf /root/.cache  \
-    &&  rm -rf /usr/lib/python2.7 \
-    &&  rm -rf /usr/lib/x86_64-linux-gnu/guile
+    && echo "==> Total size BEFORE cleanup: $(du -sh --exclude=/proc /)" \
+    && echo "Removing: apt caches" && rm -rf /var/lib/apt/lists/* \
+    && echo "Removing: pip cache" && rm -rf /root/.cache  \
+    && echo "Removing: pycache files" &&  find / -regex '^.*\(__pycache__\|\.py[co]\)$' -delete \
```

Comment:
Removing pycache files [can in some cases be helpful](https://stackoverflow.com/a/59786706/13238); and saves ~20MB